rest-endpoint/src/main/scala/za/co/absa/spline/ol/rest/RootServlet.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.ol.rest
+
+import org.scalatra.ScalatraServlet
+
+class RootServlet extends ScalatraServlet {
+  get("/") {
+    contentType="text/html"
+
+    <html lang="en">
+      <head>
+        <link href={url("/assets/css/bootstrap.css")} rel="stylesheet" />
+      </head>
+      <body>
+        <div class="container">
+          <div class="page-header">
+            <h1>Spline Open Lineage Kafka Proxy</h1>
+          </div>
+
+          <div class="row">
+            <div class="span12">
+              <h3>Open Lineage API Endpoint</h3>
+              <pre>{serverHost + ":" + serverPort + url("/lineage")}</pre>
```

Comment:
can it be a string interpolation?
```suggestion
              <pre>{s"$serverHost:$serverPort${url("/lineage")}"}</pre>
```